mne/beamformer/_dics.py

```diff
@@ -162,7 +166,8 @@ def make_dics(info, forward, csd, reg=0.05, noise_csd=None, label=None,
     .. footbibliography::
     """  # noqa: E501
     rank = _check_rank(rank)
-    _check_option('pick_ori', pick_ori, [None, 'normal', 'max-power'])
+    _check_option('pick_ori', pick_ori,
+                  [None, 'vector', 'normal', 'max-power'])
```

Comment:
Again, should this not be #10980 ?

Reply:
Same as above, will refactor out once merged but otherwise, the tutorial won't run since it uses the most general case (epochs + vector + tfr).